pkg/api/v1/client/client.go

```diff
@@ -188,6 +199,14 @@ func (c *APIClient) executeRequest(ctx context.Context, method, endpoint string,
 	return c.doRequest(agent, response)
 }
 
+// RPCResponseWrapper is a wrapper for RPC responses
+type RPCResponseWrapper struct {
```

Comment:
[non-blocking] isn't there already an RPC response type? What is different about this one?

Reply:
The difference is just the data type for client vs server side. However, I think this is redundant and will remove that wrapper. Nice catch btw ;)